### ISSUE TITLE
Hard Terminal Hang on compile_month

### ISSUE THREAD

**Issue by CalebRHicks** [tomerikhaugen/LaTeX_LabBook#12]
Terminal hangs and must be force-closed when compile_month is called in a different directory.

**Comment by CalebRHicks** [comment]
May not be caused by different directory, may be an issue when running on the HPCC

**Comment by tomerikhaugen** [comment]
You are correct as it stands compile_month only works when run in the LabBook directory
